crates/aptos-genesis/src/config.rs

```diff
@@ -103,6 +107,7 @@ impl Default for Layout {
             total_supply: None,
             employee_vesting_start: Some(1663456089),
             employee_vesting_period_duration: Some(5 * 60), // 5 minutes
+            on_chain_consensus_config: OnChainConsensusConfig::default(),
```

Comment:
Can you add support for `OnChainExecutionConfig` as well ? :) that way we can just remove some ugly overrides @msmouse has for large TPS test to run...

Reply:
Sure, done.

Reply:
Thanks bosses!